TramsDataApi/UseCases/IndexAcademyTransferProjects.cs

```diff
@@ -20,14 +20,42 @@ public class IndexAcademyTransferProjects : IIndexAcademyTransferProjects
             _trustGateway = trustGateway;
         }
 
-        public IList<AcademyTransferProjectSummaryResponse> Execute(int page)
+        public Tuple<IList<AcademyTransferProjectSummaryResponse>, int> Execute(int page)
         {
-            var listOfAcademyTransferProjects = _academyTransferProjectGateway.IndexAcademyTransferProjects(page);
+           Tuple<IList<AcademyTransferProjects>, int> listOfAcademyTransferProjects =
+              _academyTransferProjectGateway.IndexAcademyTransferProjects(page);
 
-            return listOfAcademyTransferProjects.ToList().Select(atp =>
-            {
-                return AcademyTransferProjectSummaryResponse(atp);
-            }).ToList();
+           return Tuple.Create<IList<AcademyTransferProjectSummaryResponse>, int>(
```

Comment:
Why can't you use the AcademyTransferProjectSummaryResponse constructor anymore?

Reply:
It does; two lines below this - I didn't change that code, just wrapped what was there into a tuple to include the total count.

Reply:
I see what you mean - Dom's change moved that code into the constructor. Missed that part of the merge conflicts.

Reply:
https://github.com/DFE-Digital/trams-data-api/commits/825e78590db4811740e2cf623b26e624282c4c56/TramsDataApi/UseCases/IndexAcademyTransferProjects.cs

Reply:
Updated to use the helper method - defintitely nicer to look at!